books/RayTracingInOneWeekend.html

```diff
@@ -45,9 +45,28 @@
 
 https://github.com/RayTracing/raytracing.github.io/
 
-I assume a little bit of familiarity with vectors (like dot product and vector addition). If you
+A note on the implementing code for these books -- our philosophy for the included code prioritizes
+the following goals:
+
+  - The code should implement the concepts covered in the books.
+
+  - We use C++, but as simple as possible. Our programming style is very C-like, but we take
+    advantage of modern features where it makes the code easier to use or understand.
+
+  - Our coding style continues the style established from the original books as much as possible,
+    for continuity.
+
+  - Line length is kept to 96 characters per line, to keep lines consistent between the codebase and
+    code listings in the books.
+
+The code thus provides a baseline implementation, with tons of improvements left for the reader to
+enjoy. There are endless ways one can optimize and modernize the code; we prioritize the simple
+solution.
+
+We assume a little bit of familiarity with vectors (like dot product and vector addition). If you
 don’t know that, do a little review. If you need that review, or to learn it for the first time,
-check out Marschner’s and my graphics text, Foley, Van Dam, _et al._, or McGuire’s graphics codex.
+check out Marschner’s and Peter's graphics text, Foley, Van Dam, _et al._, or McGuire’s graphics
+codex.
```

Comment:
Perhaps include ISBNs and a link to the codex?

Not necessary I think

Reply:
Changed to:

> We assume a little bit of familiarity with vectors (like dot product and vector addition). If you
> don’t know that, do a little review. If you need that review, or to learn it for the first time,
> check out the online [_Graphics Codex_][gfx-codex] by Morgan McGuire, _Fundamentals of Computer
> Graphics_ by Steve Marschner and Peter Shirley, or _Fundamentals of Interactive Computer Graphics_
> by J.D. Foley and Andy Van Dam.